jetty-spring/src/main/java/org/eclipse/jetty/spring/SpringConfigurationProcessor.java

```diff
@@ -68,6 +68,45 @@
     private DefaultListableBeanFactory _beanFactory;
     private String _main;
 
+    @Override
+    public void init(org.eclipse.jetty.util.resource.Resource jettyResource, XmlParser.Node config, XmlConfiguration configuration)
```

Comment:
It's not clear to me why we need this overloaded method. Since a `Resource` can always be turned into a `URL`, can't we just use the older `init(URL, ...)` method?

Reply:
The `ConfigurationProcessor` interface is used by groups outside of Jetty (namely Spring).
We can clean this up for Jetty 10.0.x, definitely.

Going back and forth between URL and Resource multiple times will lose escaping, especially on Windows.
If we can stick with Resource all the way, and never work with URL the better. (In fact, this is the way that Jetty 10.x should be tweaked to work)

Reply:
I have removed the URL version.